### PR TITLE
Vue Updates

### LINKED ISSUE #3607: Performance content for Vue is Confusing
## Issue

This an issue with:

Platform/SDK documentation (Which one?)
Vue

## Description

The Set Up page for performance monitoring is displaying JavaScript snippets rather than the content created for Vue.

### Suggested Solution

Currently, Vue doesn't use the common content for the Getting Started page or Performance. To fix this:
1. The "Monitor Performance" section needs to be consumed as part of the Getting Started content (so you set both up, rather than one, then the other)
2. The Set Up page for Performance Monitoring should use Vue snippets (build out the include directories that the Set Up page consumes)

### PR DESCRIPTION
**Pull request by onurtemizkan**
Fixes: #3607 

- [x] Merge `Monitor Performance` with standard configuration.
- [x] Move Vue-specific snippets inside `includes/performance/...` and use from there.
- [x] Reference to `@sentry/vue` instead of `@sentry/browser` from Vue guide page.

### CODE REVIEW

**Comment by onurtemizkan** [comment]
@PeloWriter, @rhcarvalho. In addition to this, the structure of the Vue guide doesn't seem to be in the standard(?) format; having `primer`, `install`, `config` ... sections, like the other platform guides. Should I update that too in this PR, as we're visiting Vue guides anyways?

**Comment by PeloWriter** [comment]
Yes, please make those changes. To fully update Vue, you'll delete its index.mdx file so only the prose from the common content displays with Vue code samples. That's described in the [Contributor Guidelines for Getting Started](https://docs.sentry.io/contributing/approach/sdk-docs/write-getting-started/).

**Comment by onurtemizkan** [comment]
@PeloWriter made the updates, it's ready for another round of review.

**Review by PeloWriter — APPROVED** [review]
Nice job here, @onurtemizkan. A few tweaks to ensure there's minimal redundancy with the common content. Other than that, this looks good to me.

File: src/includes/getting-started-verify/javascript.vue.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Add a button to your view template that throws an error:
```
[the remark this answers is not in the document]
Comment (rhcarvalho) [reply src/includes/getting-started-verify/javascript.vue.mdx:1]: Yeah, nice addition!

Q: is it `view template` or `Vue template` here?

**Review by rhcarvalho — APPROVED** [review]
Only had a high-level pass, seems fine. Thanks @onurtemizkan!

File: src/includes/getting-started-verify/javascript.vue.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (onurtemizkan) [reply src/includes/getting-started-verify/javascript.vue.mdx:1]: I'm not very familiar with Vue terminology 😅  but just thought this as "one of the view templates of a Vue app"

[the remark this answers is not in the document]
Comment (rhcarvalho) [reply src/includes/getting-started-verify/javascript.vue.mdx:1]: Ah no worries. Would this avoid the potential for confusion?

```suggestion
Add a button to your page that throws an error:
```

[the remark this answers is not in the document]
Comment (onurtemizkan) [reply src/includes/getting-started-verify/javascript.vue.mdx:1]: I think so, thanks :)

**Pull request merged**